### DISCUSSION TITLE
Why are there two hotkeys for selection: x & d?

### DISCUSSION THREAD

**Discussion by badarotti** [Byron/dua-cli#164] [Q&A]
Just learned about dua from the [Slef-Hosted podcast n.98](https://selfhosted.show/98). I love it, thanks for the great work. Also from the discussion here I found out about XPLR. 

What is the rational about having 2 methods of selecting files (d & x )  with x not being togable. I found this a bit confusing, especially since there's only one operation to be done on the selected files.

Thanks.

**Comment by Byron** [comment]
Thank you! Apparently I starred [`XPLR`](https://github.com/sayanarijit/xplr) already, but totally forgot about it even though it seems like an awesome utility, too!

I actually don't recall if this was a feature request or if it was something I felt was useful. However, when playing with it now I was confused that the inverse operation of `x`, unconditionally *deselect* something, wasn't implemented yet. With that I'd feel that `x` and `X` would be siblings of `d`, one sects and the other deselects. In any case, one might wonder if it's useful in practice and I'd probably say "no" most of the time.

Maybe the answer is buried in the `git` history somewhere so we see if `x` should be removed, or get a sibling `X` for symmetry.
